R/02-downAndUploadModel.R

```diff
@@ -127,7 +127,11 @@ uploadModel <- function(input, output, session, values, model, uploadedNotes){
       uploadedNotes(readLines("README.txt"))
     })
     if (inherits(res, "try-error")) {
-      shinyjs::alert("Could not load file.")
+      shinyjs::alert(paste0("Could not load file.\n",
+                            "The file to be uploaded should be a .zip file",
+                            " that contains the following files:",
+                            " \n help.html,\n model.rds,\n README.txt\n",
```

Comment:
I just made a .zip file as well with all the README and help.html files, but I was wondering if the mode must be RDS format? I uploaded the zip file with .Rdata format and it worked. I think its okay to say here:
`\n model.Rdata` ? what do you think?

Reply:
Yes, I could reproduce this although I do not understand why the command
`modelImport <- readRDS("model.rds")`
imports the `*.RData` file even under the correct name `modelImport` **locally** so that all looks fine.

We need to discuss here how to best proceed here.